src/styles/_fontMixin.scss

```diff
@@ -0,0 +1,5 @@
+@mixin fontSize($w, $s, $h) {
```

Comment:
as i see there is  not only `Size` rename plz to `font` or `fontStyle`

Reply:
fixed